terraform/environments/wardship/load_balancer.tf

```diff
@@ -231,17 +233,21 @@ resource "aws_security_group" "lb_sc_pingdom_2" {
 }
 
 
+#tfsec:ignore:AVD-AWS-0053
 resource "aws_lb" "wardship_lb" {
+  #checkov:skip=CKV_AWS_91: "ELB Logging not required"
   name                       = "wardship-load-balancer"
   load_balancer_type         = "application"
   security_groups            = [aws_security_group.wardship_lb_sc.id, aws_security_group.lb_sc_pingdom.id, aws_security_group.lb_sc_pingdom_2.id]
   subnets                    = data.aws_subnets.shared-public.ids
-  enable_deletion_protection = false
+  enable_deletion_protection = true
```

Comment:
I think we need to leave this as false, otherwise terraform can't remove it, if it needs to be recreated.